File: Entities/Models/CharacterItem.cs
namespace Entities.Models;

public class CharacterItem
{
    public int ItemId { get; set; }
    public Item Item { get; set; } = null!;
    public int CharacterId { get; set; }
    public Character Character { get; set; } = null!;
    public int ItemNumber { get; set; } = 0;
    public float CurrentDurability { get; set; } = 1.0f;
}

File: Client/src/app/model/characterItem.ts
export class CharacterItem {
    characterId:number;
    itemId:number;
    itemNumber:number;
    currentDurability:number;

    constructor(
        characterId:number, 
        itemId:number, 
        number:number, 
        currentDurability:number) {
        this.characterId = characterId;
        this.itemId = itemId;
        this.itemNumber = number;        
        this.currentDurability = currentDurability;
    }
}
